src/components/calcite-checkbox/calcite-checkbox.scss

```diff
@@ -58,9 +58,7 @@
   }
```

Comment:
Possibly a question for @asangma - it looks like this mixin is only used once, should we kill? Create a util for this?